src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -696,4 +698,97 @@ public void setApplicationProtocols(String[] protocols) {
         }
         applicationProtocols = tempProtocols;
     }
+
+    /**
+     * Returns a prioritized array of signature scheme names that can be used
+     * over the SSL/TLS/DTLS protocols.
+     * <p>
+     * Note that the standard list of signature scheme names may be found in
+     * the <a href=
+     * "{@docRoot}/../specs/security/standard-names.html#signature-schemes">
+     * Signature Schemes</a> section of the Java Security Standard Algorithm
+     * Names Specification. Providers may support signature schemes not found
+     * in this list or might not use the recommended name for a certain
+     * signature scheme.
+     * <p>
+     * The returned array could be {@code null}, in which case the underlying
+     * provider-specific default signature schemes will be used over the
+     * SSL/TLS/DTLS connections.
+     * <p>
+     * The returned array could be empty (zero-length), in which case the
+     * signature scheme negotiation mechanism is turned off for SSL/TLS/DTLS
+     * protocols, and the connections may not be able estabilished if the
+     * negotiation mechanism is required by a certain SSL/TLS/DTLS protocol.
+     * <p>
+     * The returned array could be neither {@code null} nor empty (zero-length),
+     * in which case the signature schemes in the returned array will be used
+     * over the SSL/TLS/DTLS connections.
+     *
+     * @implSpec
+     * For non-null returns, this method will return a new array each time it
+     * is invoked. Providers should ignore unknown signature scheme names
+     * while establishing the SSL/TLS/DTLS connections.
+     *
+     * @implNote
+     * Note that the underlying provider may define the default signature
+     * schemes for each SSL/TLS/DTLS connection.  Applications may also use
+     * System Property, {@systemProperty jdk.tls.client.SignatureSchemes}
+     * and/or {@systemProperty jdk.tls.server.SignatureSchemes}, to customize
+     * the provider-specific default signature schemes. If the
+     * {@link #setSignatureSchemes} method has not been called, this method
+     * should return the default signature schemes for connection populated
+     * objects, or {@code null} for pre-populated objects.
+     *
+     * @return {@code null} or an array of signature scheme {@code String}s.
```

Comment:
Suggest rephrasing as "an array of signature scheme {@code Strings} or {@null} if none have been set."